tests/split-store.bats

```diff
@@ -89,9 +89,6 @@ load helpers
 	[[ "$output" =~ "Name: $name" ]]
 	[[ "$output" =~ "Data: random1" ]]
 	[[ "$output" =~ "Data: random2" ]]
-	# Since this image is being read from the readonly graph root
-	# so it must show that
-	[[ "$output" =~ "Read Only: true" ]]
```

Comment:
Could we test it is writeable instead?

Reply:
Modified the test to verify if we can actually delete image from `graphRoot` while `--image-store` is still set.